src/views/Tasks/ViewTask/index.jsx

```diff
@@ -690,7 +696,13 @@ export default class ViewTask extends Component {
     return (
       <Dashboard
         helpView={<HelpView description={description} />}
-        search={<Search onSubmit={this.handleTaskSearchSubmit} />}>
+        search={
+          <Search
+            value={value}
+            onChange={this.handleTaskSearchChange}
+            onSubmit={this.handleTaskSearchSubmit}
```

Comment:
I just realized that we can accomplish this via:

```jsx
defaultValue={this.props.match.params.taskId}
```

Reply:
Yay Over-engineering 😅